exchanges/bitfinex/bitfinex_wrapper.go

```diff
@@ -773,7 +797,97 @@ func (b *Bitfinex) ValidateCredentials() error {
 	return b.CheckTransientError(err)
 }
 
+// FormatExchangeKlineInterval returns Interval to exchange formatted string
+func (b *Bitfinex) FormatExchangeKlineInterval(in kline.Interval) string {
```

Comment:
Due to the sensitivity of this, I think it would be wise to add tests to make sure that these values match up

Reply:
👍 will add test coverage to exchanges